lib/msal-browser/src/index.ts

```diff
@@ -48,8 +48,12 @@ export {
     AuthenticationResult,
     // Error
     InteractionRequiredAuthError,
+    InteractionRequiredAuthErrorMessage,
     AuthError,
     AuthErrorMessage,
+    ClientAuthError,
+    ClientAuthErrorMessage,
+    ServerError,
```

Comment:
I am not sure if [the `ClientConfigurationError` error class](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/error/ClientConfigurationError.ts) should also be exported. For now I'm assuming it's an internal error and not exporting it.

Reply:
Yes it should also be exported. Can we also have these exported from `msal-node`, if not already?

Reply:
Updated.